src/test/java/podman/client/PodmanClientContainersTest.java

```diff
@@ -73,6 +74,20 @@ void createAndDeleteContainer() throws Throwable {
         assertThat(awaitResult(client.containers().exists(id))).isFalse();
     }
 
+    @Test
+    void instpectContainer() throws Throwable {
+        JsonObject createResult = awaitResult(
+                client.containers().create(new ContainerCreateOptions().image("quay.io/podman/hello:latest")));
+        String id = createResult.getString("Id");
+
+        awaitResult(client.containers().start(id));
+
+        JsonObject inspection =
+                awaitResult(client.containers().inspect(id, new ContainerInspectOptions().setSize(false)));
+        assertThat(inspection.containsKey("Id")).isTrue();
+        assertThat(inspection.getString("Id")).isEqualTo(id);
```

Comment:
We should remove the container upon completion, unless it's by default (I can't remember)